tests/config/selfupdate.toml

```diff
@@ -16,4 +16,5 @@ primary_ecu_serial = "723f79763eda1c753ce565c16862c79acdde32eb922d6662f088083c51
 primary_ecu_hardware_id = "selfupdate-device"
 
 [pacman]
-type = "debian"
+type = "none"
```

Comment:
Not a big deal or even necessary, but we might consider renaming these `selfupdate` bits if they no longer with be actually self-updating.

Reply:
So what might be a better name to replace ‘selfupdate'? fakeupdate or testupdate?

Reply:
`testupdate` is okay. I'm trying to think of something better, but I mostly just want to get `self` out of there since it isn't accurate, and `update` on it's own is obviously too broad. :)